src/app/container/email.service.ts

```diff
@@ -76,7 +76,10 @@ export class EmailService {
    * @param tool The tool to contact author for
    */
   public composeContactAuthorEmail(tool: DockstoreTool): string {
-    const email = EmailService.getInquiryEmailMailTo(tool.email);
+    let email: string = null;
+    if (tool.authors && tool.authors.length) {
```

Comment:
Getting into edge case territory, especially because I think we only have one tool with a private Docker image, but it doesn't look like all authors have emails, at least not in the DB. So ideally you'd check for the first author with an email, and if there aren't any not return anything.

But it's rare enough and was probably already like this, so it could be a followup ticket.

Reply:
https://github.com/dockstore/dockstore/issues/5771